PocketKit/Sources/PocketKit/Settings/SettingsView.swift

```diff
@@ -41,22 +41,17 @@ struct SettingsView: View {
 struct SettingsForm: View {
     @ObservedObject
     var model: AccountViewModel
+
     var body: some View {
         Form {
             Group {
-                //                Section(header: Text("Your Account").style(.settings.header)) {
-                //                    PremiumRow(status: .notSubscribed, destination: EmptyView())
-                //                    SettingsRowLink(title: "Reset Password", destination: EmptyView())
-                //                    SettingsRowLink(title: "Delete Account", destination: EmptyView())
-                //                }.textCase(nil)
-
```

Comment:
should we keep this uncommented instead of deleting to provide a starting point for the settings done in the future? our is this no longer applicable

Reply:
this is no longer applicable.  we already have a baseline for settings